src/vcCamera.cpp

```diff
@@ -543,6 +547,8 @@ void vcCamera_SwapMapMode(vcState *pProgramState)
     // defer actually swapping projection mode
     pProgramState->cameraInput.transitioningToMapMode = true;
     lookAtPosition += udDouble3::create(0, 0, -1); // up
+
+    cameraHeight = pProgramState->settings.camera.farPlane;
```

Comment:
This will mean anything below 0 won't be visible

Reply:
When @peterAdams1014 added map mode it worked well with auto-selecting a height. Might be worthwhile going through the tree and seeing what he did originally.

Reply:
It was farplane / 2

Reply:
I think at some point I changed it to fix a bug thinking that whatever height it was at would suffice, incidentally these changes fix the original bug too, that if you swapped map mode right near the end of the transition it would recalculate orthographic size